core/server/web/site/app.js

```diff
@@ -8,30 +8,10 @@ const apps = require('../../services/apps');
 const constants = require('../../lib/constants');
 const storage = require('../../adapters/storage');
 const urlService = require('../../services/url');
-
-// This should probably be an internal app
 const sitemapHandler = require('../../data/xml/sitemap/handler');
-
-// Route Service
+const themeService = require('../../services/themes');
```

Comment:
Just wondering if this change should go in this PR, as its not related to shared middlewares from web/ and seems a bit odd one out. I get how this is a similar change, but then there will be more similar as well and then all of them can go as separate refactor maybe :)

Reply:
Ok 👍 Will make 2 commits

Reply:
I'll just undo.